<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassSchedule</class>
 <widget class="QWidget" name="ClassSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>显示已结课</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_flash">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_edit">
        <property name="text">
         <string>编辑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>学员列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView_stu"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>回收站</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView_trash"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_classTable" native="true">
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
